system-int-test/test/cmr/system_int_test/bootstrap/bulk_index_test.clj

```diff
@@ -690,3 +694,85 @@
       (testing "All tag parameters with XML references"
         (is (d/refs-match? [coll1]
                            (search/find-refs :collection {:tag-key "tag1"})))))))
+
+(deftest bulk-index-variables-for-provider
```

Comment:
This test file is getting too big. How about putting the bulk indexing variables tests in its own namespace?